.ci/azure/linux.yml

```diff
@@ -516,6 +516,11 @@ jobs:
       $(RUN_PREFIX) python3 -m pytest $(LAYER_TESTS_DIR)/mo_python_api_tests/test_mo_convert_pytorch.py --ir_version=11 --junitxml=./TEST-test_mo_convert_pytorch.xmlTEST
     displayName: 'MO Python API Tests - Import PyTorch model from memory'
 
+  - script: |
+      python3 -m pip install -r $(LAYER_TESTS_DIR)/requirements.txt
+      $(RUN_PREFIX) python3 -m pytest $(LAYER_TESTS_DIR)/py_frontend_tests --junitxml=./TEST-test_mo_convert_pytorch.xmlTEST
```

Comment:
this step uses the same file name as previous one `TEST-test_mo_convert_pytorch.xmlTEST`. Please, select different name
Secondly, is file extension `xmlTEST` correct?

Reply:
I believe extension can be any, but yeah seems weird, will change to xml